indico/modules/events/registration/__init__.py

```diff
@@ -77,7 +75,7 @@ def _inject_event_header(event, **kwargs):
                 .order_by(db.func.lower(RegistrationForm.title))
                 .all())
 
-    registrations = get_published_registrations(event)
+    registrations = event.published_registrations
```

Comment:
No, I couldn't. That's injected by the registration module, as a timetable hook.